.github/workflows/ci.yml

```diff
@@ -407,7 +408,7 @@ jobs:
           script: |
             echo $GITHUB_REPOSITORY
             adb devices
-            ./gradlew connectedCheck
+            ./gradlew app:connectedCheck
```

Comment:
In case we run `./gradlew connectedCheck` CI will run Integration tests on each module, and in case there is no test it will fail, and CI will complain. So we just have to run Integration tests inside `app`